tests/unit/test_models.py

```diff
@@ -37,14 +37,3 @@ def test_multiple_choice_response_count():
 
     for answer in answers:
         factories.AnswerFactory(question=question, multiple_choice_responses=answer)
-
```

Comment:
What happened to this? Is this tested somewhere else?

Reply:
We didn't need the function in the model anymore so I removed it, and therefore removed the test!

Reply:
That is a good reason - missed that the function had been removed!